# Sending the signs
---
std_msgs/Float64MultiArray signs


